File: Web/RMA-Web-Shell/projects/clientcare/src/app/policy-manager/shared/entities/role-player-policy-online-submission-detail.ts
import { CategoryInsuredEnum } from "../enums/categoryInsuredEnum";

export class RolePlayerPolicyOnlineSubmissionDetail {
  rolePlayerPolicyOnlineSubmissionDetailId: number;
  rolePlayerPolicyOnlineSubmissionId: number;
  productOptionId: number;
  categoryInsured: CategoryInsuredEnum
  averageNumberOfEmployees: number;
  averageEmployeeEarnings: number;
  liveInAllowance: number;
  isDeleted: boolean;
  createdBy: string;
  createdDate: Date;
  modifiedBy: string;
  modifiedDate: Date;
}

File: Service/ClientCare/RMA.Service.ClientCare.Contracts/Entities/Member/RolePlayerPolicyOnlineSubmissionDetail.cs
using RMA.Service.Admin.MasterDataManager.Contracts.Enums;

namespace RMA.Service.ClientCare.Contracts.Entities.Member
{
    public class RolePlayerPolicyOnlineSubmissionDetail
    {
        public int RolePlayerPolicyOnlineSubmissionDetailId { get; set; } // RolePlayerPolicyOnlineSubmissionDetailId (Primary key)
        public int RolePlayerPolicyOnlineSubmissionId { get; set; } // RolePlayerPolicyOnlineSubmissionId
        public int ProductOptionId { get; set; } // ProductOptionId
        public CategoryInsuredEnum CategoryInsured { get; set; } // CategoryInsuredId
        public int AverageNumberOfEmployees { get; set; } // AverageNumberOfEmployees
        public decimal AverageEmployeeEarnings { get; set; } // AverageEmployeeEarnings
        public decimal? LiveInAllowance { get; set; } // LiveInAllowance
        public bool IsDeleted { get; set; } // IsDeleted
        public string CreatedBy { get; set; } // CreatedBy (length: 50)
        public System.DateTime CreatedDate { get; set; } // CreatedDate
        public string ModifiedBy { get; set; } // ModifiedBy (length: 50)
        public System.DateTime ModifiedDate { get; set; } // ModifiedDate
    }
}